mobile/lib/components/newest_article/newest_article_widget.dart

```diff
@@ -42,16 +42,11 @@ class NewestArticleWidget extends HookWidget {
 
     return SwipeableCard(
       items: articles.value.mapIndexed(
```

Comment:
I know it's not really targeting your changes, but we might as well address this now. Seems like mapIndexed is not necessary here, a simple map would be sufficient - unless we're planning to use index value for something in the near future. This of course requires removing index parameter in the next line.
```suggestion
      items: articles.value.map(
```

Reply:
yup, I think I overlooked it when refactoring